setup.py

```diff
@@ -0,0 +1,48 @@
+import os
+from configparser import ConfigParser
+
+from requests import get
+
+from core.utils.check import check_ip_address
+from core.wg.keygen import generate_private_key, generate_public_key
+
+
+def make_config(path):
+    #Create pair of crypto keys
+    server_private_key = generate_private_key()
+    server_public_key = generate_public_key(server_private_key)
+
+    #Get external ip of your server
+    external_ip = get('https://api.ipify.org').content.decode('utf8')
+    tg_token = input("[ ! ] Telegram bot token: ")
+    admins = input("[ ! ] Admin Telegram IDs with ',' separator: ")
+    ip_range = get_ip_range()
+    endpoint_port = get_endpoint_port()
+
+    #Put all data into config
+    config = ConfigParser()
+    config.optionxform = str
+    config["TelegramBot"] = {"token":f"{tg_token}", "admins": f"{admins}"}
+    config["db"] = {"path": "db.sqlite"}
+    config["Server"] = {"IP":f"{ip_range}", "PrivateKey": f"{server_private_key}", "PublicKey": f"{server_public_key}", "EndpointIP": f"{external_ip}", "EndpointPort":f"{endpoint_port}"}
```

Comment:
```suggestion
    config["Server"] = {
    	"IP": ip_range, 
    	"PrivateKey": server_private_key, 
    	"PublicKey": server_public_key, 
    	"EndpointIP": external_ip, 
    	"EndpointPort": endpoint_port
    }
```
unnecessary f-strings. and prettify to be more readable